sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -116,7 +116,7 @@ Recognize data from US sales receipts using a prebuilt model.
 ```C# Snippet:FormRecognizerSampleRecognizeReceiptFileStream
 using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
 {
-    Response<IReadOnlyList<RecognizedReceipt>> receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
+    Response<RecognizedReceiptCollection> receipts = await client.StartRecognizeReceipts(stream).WaitForCompletionAsync();
```

Comment:
can we do implicit cast here now that it is possible?
Although not in the guidelines, it has been the recommendation from Krzysztof and it will reflect what other libraries do